nexus_constructor/model/stream.py

```diff
@@ -0,0 +1,47 @@
+import attr
+
+from nexus_constructor.writer_modules import WriterModules
```

Comment:
I really should not point out that this file feels a bit incomplete as it is very obvious. 😃